test/auditing.jl

```diff
@@ -332,6 +332,59 @@ end
     end
 end
 
+@testset "Auditor - relocatable pkg-config prefix" begin
+    for os in ["linux", "macos", "freebsd", "windows"]
+        platform = Platform("x86_64", os)
+        mktempdir() do build_path
+            build_output_meta = nothing
+            @test_logs (:info, r"Relocatize pkg-config file .*/destdir/lib/pkgconfig/libfoo.pc$") match_mode=:any begin
+                build_output_meta = autobuild(
+                    build_path,
+                    "libfoo",
+                    v"1.0.0",
+                    # Copy in the libfoo sources
+                    [DirectorySource(build_tests_dir)],
+                    # Build libfoo using autotools to create a real .la file, and also
+                    # create a fake .la file (which should not be removed).  Create also a
+                    # symlink libqux.la -> libfoo.la, which will be broken after libfoo.la
+                    # has been deleted: remove libqux.la as well
+                    libfoo_autotools_script * raw"""
+                    touch ${prefix}/lib/libbar.la
+                    ln -s ${prefix}/lib/libfoo.la ${prefix}/lib/libqux.la
+                    """,
```

Comment:
Sounds like this was copied from the test above but it's unrelated to the pkgconfig test, no?
```suggestion
                    libfoo_autotools_script,
```